src/aiortc/contrib/media.py

```diff
@@ -108,8 +116,10 @@ def player_worker(
                 time.sleep(0.01)
                 continue
             if audio_track:
+                print("Put None in audio in player_worker")
```

Comment:
Do you want to keep these as prints for a specific reason? Because this is an error case? if so, would also add it to logs (like have both a print and a log so that it shows up in logs too)

Reply:
changed all the prints to logs